test/calendar.test.tsx

```diff
@@ -9,14 +9,16 @@ const CalendarWithUtils = getCalendar(new DateFnsUtils());
 
 export default test("Calendar")
   .step(Page.visit("/"))
-  .child("filter by locator", (test) => test.step(renderComponent()).assertion(Calendar("18 August 2014").exists()))
-  .child("filter by title", (test) =>
-    test.step(renderComponent()).assertion(Calendar({ title: "August 2014" }).exists())
+  .child("filters", (test) =>
+    test
+      .step(renderComponent())
+      .assertion(Calendar("18 August 2014").exists())
+      .assertion(Calendar({ title: "August 2014" }).exists())
+      .assertion(Calendar({ month: "August" }).exists())
+      .assertion(Calendar({ year: "2014" }).exists())
+      .assertion(Calendar({ day: 18 }).exists())
+      .assertion(Calendar({ weekDay: "Mo" }).exists())
```

Comment:
Since all the tests in this file use the same locator, I would assign it once at the top:

```ts
const calendar = Calendar("18 August 2014");
```
and then use the assertion form for things like filters:

```ts
assertion(calendar.has({ title: "August 2014" }));
```